src/app/admin/basePage/basePage.component.html

```diff
@@ -15,19 +15,19 @@ <h1>Base Icon List</h1>
       <div *ngIf="baseIcons.length > 0">
         <div class="card" *ngFor="let baseIcon of baseIcons">
           <div class="card-body p-2" *ngIf="baseIcon.icon">
-            <svg viewBox="0 0 24 24" style="width: 1.5rem;height: 1.5rem">
+            <svg viewBox="0 0 24 24" style="width: 1.5rem;height: 1.5rem;margin-right: 0.1rem">
```

Comment:
Is this the way to go? Increasing the width seems to work as well, but certain values make the icons blurry in Firefox because it is pushed off-grid by sub-pixel amounts...

Reply:
Probably could have added it to the parent. Yeah, firefox is annoying with SVGs.